tests/phpunit/tests/oembed/template.php

```diff
@@ -345,4 +345,60 @@ public function test_wp_maybe_enqueue_oembed_host_js_without_wp_head_action() {
 		wp_maybe_enqueue_oembed_host_js( $post_embed );
 		$this->assertFalse( $scripts->query( 'wp-embed', 'enqueued' ) );
 	}
+
+	/**
+	 * @ticket 35567
+	 */
+	public function test_is_embeddable_post_non_existent_post() {
+		$this->assertFalse( is_post_embeddable( 0 ) );
```

Comment:
Technically this will be checking the global `$post`. This is more descriptive:

```suggestion
		$this->assertFalse( is_post_embeddable( 99999 ) );
```